api/controllers/v1/massif/get-statistics.js

```diff
@@ -0,0 +1,79 @@
+const ControllerService = require('../../../services/ControllerService');
+const DashboardDataMassifService = require('../../../services/StatisticsMassifService');
+const ErrorService = require('../../../services/ErrorService');
+
+module.exports = async (req, res) => {
+  try {
+    const params = {
+      controllerMethod: 'MassifController.get-statistics',
+    };
+
+    if (!req.params.id) {
+      return ControllerService.treat(
+        req,
+        'Missing massif id',
+        null,
+        params,
+        res
+      );
+    }
+
+    await DashboardDataMassifService.createMaterializedView();
+
+    const caveWithMaxDepth =
+      DashboardDataMassifService.getCaveWithMaxDepthInMassif(req.params.id);
+
+    const nbCaves = DashboardDataMassifService.getNbCavesInMassif(
+      req.params.id
+    );
+
+    const nbNetworks = DashboardDataMassifService.getNbNetworksInMassif(
+      req.params.id
+    );
+
+    const caveWithMaxLength =
+      DashboardDataMassifService.getCaveWithMaxLengthInMassif(req.params.id);
+
+    const nbCavesWhichAreDiving =
+      DashboardDataMassifService.getNbCavesWhichAreDivingInMassif(
+        req.params.id
+      );
+
+    const avgDepthAndLength =
+      DashboardDataMassifService.getAvgDepthAndLengthInMassif(req.params.id);
+
+    const totalLength = DashboardDataMassifService.getTotalLength(
+      req.params.id
+    );
+
+    return Promise.all([
+      nbCaves,
+      nbNetworks,
+      caveWithMaxDepth,
+      caveWithMaxLength,
+      nbCavesWhichAreDiving,
+      avgDepthAndLength,
+      totalLength,
+    ]).then((values) => {
+      const data = {
+        nb_caves: Number.parseInt(values[0].nb_caves, 10),
```

Comment:
`Number.parseInt(null, 10)` will return NaN and some of your function above may return null....
Is it what you expect ?

Reply:
If our first result is `null`, we want that the value stays on `null` since it's the front that will test it out.

Reply:
It will not be `null` but `NaN` returned by `Number.parseInt(null, 10)`. Is it what you want ?